src/modals/Deploy.js

```diff
@@ -108,10 +108,11 @@ function Deploy(props) {
     }
 
     async function handleDeploySmartWalletButtonClick() {
-        deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
+        // deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
```

Comment:
sorry, why was this commented out? do we not need it because of state setting? if so, we can just remove the line